### PR TITLE
[Install-Guide] Fixes and improvements

### PR DESCRIPTION
**Pull request by black-dragon74**
Also, cleaned up the syntax.

Signed-off-by: black.dragon74 <nickk.2974@gmail.com>

### CODE REVIEW

**Comment by black-dragon74** [comment]
Reviewer's Note: This patch also hides copy code button on the last code block in `Configure.md`. That feature depends on #732 so it will be effective once that is merged.

The annotation used to hide that button is backwards compatible and won't break any syntax highlighting/formatting on the code block.

Regards

**Comment by black-dragon74** [comment]
CC: @Shwetha-Acharya

File: docs/Install-Guide/Configure.md [hunk 1]
```diff
@@ -115,14 +115,12 @@ Brick3: node03.yourdomain.net:/export/sdb1/brick
 ```
 
 This shows us essentially what we just specified during the volume
-creation. The one this to mention is the `Status`. A status of `Created`
-means that the volume has been created, but hasn’t yet been started,
-which would cause any attempt to mount the volume fail.
+creation. The one key output worth noticing is `Status`.
+A status of `Created` means that the volume has been created,
+but hasn’t yet been started, which would cause any attempt to mount the volume fail.
```
Comment (Sheetalpamecha) [inline docs/Install-Guide/Configure.md:120]: The above code and this para is not correctly formatted. You might want to recheck it.

File: docs/Install-Guide/Configure.md [hunk 2]
```diff
@@ -115,14 +115,12 @@ Brick3: node03.yourdomain.net:/export/sdb1/brick
...
 
-Now, we should start the volume.
+Now, we should start the volume before we try to mount it.
 
 ```
-# gluster volume start gv0
+gluster volume start gv0
```
Comment (Sheetalpamecha) [inline docs/Install-Guide/Configure.md:125]: Should be ```console in line 124

**Review by Sheetalpamecha — CHANGES_REQUESTED** [review]

**Review by rakshithakamath94 — APPROVED** [review]

**Pull request merged**